lib/brain.js

```diff
@@ -2409,7 +2409,9 @@ Brain.prototype._chooseFiat = function _chooseFiat () {
   this.tx = Tx.update(this.tx, {fiatCode: this.fiatCode, direction: 'cashOut'})
 
   const response = this._getFiatButtonResponse()
-  if (response.activeDenominations.isEmpty) return this._timedState('outOfCash')
+
+  // No need to check available cash on redeem. It is already reserved on l-s.
+  if (response.activeDenominations.isEmpty && !this.redeem) return this._timedState('outOfCash')
```

Comment:
Hmm, this is simple to implement but shouldn't we be fixing the original problem that we were double counting instead of giving up on the check entirely? I'm not sure we should completely rely on the l-s reservation.